archetypes/embedded/src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,105 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>\${groupId}</groupId>
+    <artifactId>\${artifactId}</artifactId>
+    <packaging>jar</packaging>
+    <version>\${version}</version>
+    <name>A sample project using Infinispan</name>
+    <url>http://www.myorganization.org</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.infinispan</groupId>
+                <artifactId>infinispan-bom</artifactId>
+                <!-- Which Infinispan version do you want to use?  Released versions are always more reliable than snapshots! -->
+                <version>${project.version}</version>
```

Comment:
do you have the generated pom.xml? is this properly replaced?

Reply:
Yep. I checked  manually, but the integration- test compiles the source generated from the archetype to make sure everything is as expected